code/game/objects/items/toys.dm

```diff
@@ -1240,6 +1241,28 @@
 		visible_message("[src] says, \"[phrase]\"")
 		playsound(src.loc, pokesound, 50, 0) // Should work for any pokesound anyone might want to add to fumos.
 
+/obj/item/toy/plushie/fumo/arcueid // Make sure all future fumos are children of /fumo and not this one.
+	name = "\improper Neco Arc fumo"
+	desc = "...What the fuck?"
+	icon_state = "arcueid"
+	phrase = "Burenyaa~"
+	pokesound = 'sound/sanity/burenyaa.ogg'
+
+/obj/item/toy/plushie/fumo/arcueid/attackby(obj/item/W as obj, mob/user as mob)
```

Comment:
As funny as it is, I would have to say no to this. The new arc should just be purchasable with a "fake" cigarette rather than suddenly altered by giving it one. It pushes the bounds a bit. -Cora

Reply:
Alright but pushing the bounds on...what, exactly?
I thought it was at least a fun gimmick to obtain a variant of it (Mind that all these can be gotten from plushie selection as they all draw children of `/obj/item/toy/plushie/`, I have never seen people "purchasing" plushies)

Reply:
Pushing the bounds of HRP itself. I would just say "realism" but PSSH this is soj. It just feels letting a plushy bum a ciggie to entirely change its form is a step in the door others could use to start pushing more and more for "silly innocent things" that if someone really wanted to as a researcher, they could question quite logically so to how the hell the plushie completely changes. As for the way of getting plushies, that's a valid point, I can poke one of the others to put plushies into their own machine or an existing one.